nsq/reader.py

```diff
@@ -358,7 +344,7 @@ def _maybe_update_rdy(self, conn):
         if self.backoff_timer.get_interval() or self.max_in_flight == 0:
             return
 
-        if conn.rdy <= 1 or conn.rdy < int(conn.last_rdy * 0.25):
+        if conn.rdy == 1:
```

Comment:
I think I understand this change but want to make sure — this is because we initially set `RDY 1` and we want to make sure we adjust to an appropriate per-connection max-in-flight, right?